README.md

```diff
@@ -1 +1,51 @@
-# TODO
+# OSP-Qualification-Reports
+
+Qualification Reports recreated with every new OSP Release
+
+## How to create evaluation reports
+
+- Create a new branch from the `create-reports` branch (for instance, `my-reports`)
+  - Define the appropriate OSP environment and tools by updating `tools.csv`
+  - Define the models and projects by updating `models.csv`
+  - Go to the Github Action: [Automated Evaluation Reports and Projects](https://github.com/Open-Systems-Pharmacology/OSP-Qualification-Reports/actions/workflows/create-reports.yaml)
+    - Click on the button __Run workflow__ 
+      - Select your branch (for instance, `my-reports`)
+      - Click on the green button __Run workflow__ 
+  
+## What to do when reports are created
+
+When the evaluation reports are created, a pull request is triggered toward the `develop` branch.
+The pull request will allow users to review the updates in the reports and adopt the new version.
+
+## Models
+
+The `models.csv` file indicates which models should be run and qualified.
+The header includes the following fields:
+
+- __Execute__: If `TRUE` run the qualification. If `FALSE`, skip the qualification.
+- __Repository name__: Name of Github OSP repository from which to get the model, e.g. `7E3-Model` for [7E3-Model](https://github.com/Open-Systems-Pharmacology/7E3-Model)
+- __Released version__: Tag version of the model, e.g. `1.0`
+- __Snapshot name__: Name of the snapshot (`.json`) file, e.g. `7E3` for [7E3.json](https://github.com/Open-Systems-Pharmacology/7E3-Model/7E3.json)
```

Comment:
Actually do we need the **snapshot name** for qualification reports at all? Where is it used?
There is no analogue to the model snapshot in evalutaion plans.

Reply:
I think this is not required. I would suggest using the **repository name** instead and remove the snapshot name column.